sync_cleared_for_campus.R

```diff
@@ -0,0 +1,179 @@
+library(tidyverse)
+library(dotenv)
+library(REDCapR)
+library(lubridate)
+library(rlang)
+
+source("functions.R")
+
+# uncomment to read data from production
+#load_dot_env('prod.env')
+
```

Comment:
```suggestion

Sys.getenv("INSTANCE")
Sys.getenv("URI")
```

I will always ask for the contents of the `INSTANCE` env var to be echoed to the screen. I am that way